docs/content/latest/deploy/multi-dc/async-replication.md

```diff
@@ -0,0 +1,127 @@
+
+    ```sh
+    ./bin/yb-admin \
+      -master_addresses <consumer_universe_master_addresses> \
+      setup_universe_replication <producer_universe_uuid> \
+        <producer_universe_master_addresses> \
+        <table_id>,[<table_id>..]
+    ```
+
+    For example:
+
+    ```sh
+    ./bin/yb-admin \
+      -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
+      setup_universe_replication e260b8b6-e89f-4505-bb8e-b31f74aa29f3 \
+        127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+        000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
+    ```
+
+
+The preceding command contains three table IDs: the first two are YSQL for the base table and index, and the third is the YCQL table. 
+
+Also, be sure to specify all master addresses for producer and consumer universes in the command.
+
+If you need to set up bidirectional replication, see instructions in provided in [Setting Up Bidirectional Replication](#setting-up-bidirectional-replication). Otherwise, proceed to [Loading Data into the Producer Universe](#loading-data-into-the-producer-universe).
+
+## Setting Up Bidirectional Replication
+
+To set up bidirectional replication, repeat the procedure descried in [Setting Up Unidirectional Replication](#setting-up-unidirectional-replication) applying the steps to the yugabyte-consumer universe. You need to set up each yugabyte-producer to consume data from yugabyte-consumer.
+
+When completed, proceed to [Loading Data](#loading-data-into-the-producer-universe).
+
+## Loading Data into the Producer Universe
+
+Once you have set up replication, load data into the producer universe as follows:
+
+- Download the YugabyteDB workload generator JAR file `yb-sample-apps.jar` from [GitHub](https://github.com/yugabyte/yb-sample-apps/releases).
+
+- Start loading data into yugabyte-producer by following examples for YSQL or YCQL:
+
+  - YSQL:
+
+  ```sh
+  java -jar yb-sample-apps.jar --workload SqlSecondaryIndex  --nodes 127.0.0.1:5433
+  ```
+
+  - YCQL:
+
+  ```sh
+  java -jar yb-sample-apps.jar --workload CassandraBatchKeyValue --nodes 127.0.0.1:9042
+  ```
+
+- For bidirectional replication, repeat the preceding step in the yugabyte-consumer universe.
+
+When completed, proceed to [Verifying Replication](#verifying-replication).
+
+## Verifying Replication
+
+You can verify whether or not replication has been successful.
+
+### Unidirectional Replication
+
+For unidirectional replication, connect to the yugabyte-consumer universe using the YSQL shell (`ysqlsh`) or the YCQL shell (`ycqlsh`), and confirm that you can see the expected records.
+
+### Bidirectional Replication
+
+For bidirectional replication, repeat the procedure described in [Unidirectional Replication](#unidirectional-replication), but pump data into yugabyte-consumer. 
+
+To avoid primary key conflict errors, keep the key space for the two universes separate.
+
+### Replication Lag
```

Comment:
hopefully @rahuldesirazu can take a pass here

Reply:
@bmatican @rahuldesirazu There's a related ticket: https://github.com/yugabyte/yugabyte-db/issues/6870
I suggest we close the current ticket after I get the platform part reviewed by @chirag-yb, and then @rahuldesirazu can review the db part at the same time when he reviews #6870 
Would this be acceptable?

Reply:
Reviewed